src/decorator/index.js

```diff
@@ -172,6 +173,19 @@ const processDeclarations = (inputDeclarations, ...rest) => {
                     endpoint,
                     meta: {
                         append: get(actionOptions, 'append'),
+                        appendKey: get(actionOptions, 'appendKey'),
+                    },
+                })(dispatch)
+            }
+
+            dispatchProps[key]['PUT'] = (params, actionOptions = {}) => {
+                const endpoint = getUrl(declaration, params)
+                return nionActions.get(dataKey, {
```

Comment:
confused by how this works here -- the put is returning a GET nion action?

Reply:
Seconded. Shouldn't this be `nionActions.put`?